keel-core/src/main/kotlin/com/netflix/spinnaker/keel/notifications/NotificationType.kt

```diff
@@ -4,5 +4,7 @@ package com.netflix.spinnaker.keel.notifications
  * All valid notifiers
  */
 enum class NotificationType {
-  UNHEALTHY_RESOURCE
+  UNHEALTHY_RESOURCE,
+  PINNED,
```

Comment:
Maybe `ARTIFACT_PINNED` and `ARTIFACT_UNPINNED` to follow the naming pattern of the resource-related ones above?